tools/chess-clang/xchesscc_wrapper.in

```diff
@@ -12,7 +12,7 @@ TARGET_AIE2_LIBDIR="@VITIS_AIE2_INCLUDE_DIR@"
 
 AIETOOLS="@VITIS_AIETOOLS_DIR@"
 
-TARGET=${1^^}
+TARGET=$(echo "$1" | tr '[a-z]' '[A-Z]')
```

Comment:
Why this?

Reply:
I'm getting the weirdest errors from this PR and this is/was desperate attempt to debug.